src/app.css

```diff
@@ -15,7 +15,9 @@
 
   body {
     @apply surface-background;
-    font-family: theme('fontFamily.primary');
+    font-family: theme('fontFamily.sans');
+    font-size: theme('fontSize.sm');
```

Comment:
One thing we could consider is adjusting our tailwind theme so this is size is `base` and remove anything smaller that we don't need 🤔

Reply:
We still have a bunch of places that use/need `text-xs`. Not quite sure how we want to handle that... I've left them as is for now.